README.md

```diff
@@ -51,7 +51,7 @@ steps:
 
 | name | description | required |
 | ----------- | ----------- | ----------- |
-| `version-file` | Specify a path to search for the unity project version text file. Useful if there are multiple projects in a single repo. | false |
+| `version-file` | Specify a path to search for the unity project version text file. Useful if there are multiple projects in a single repo. Pass `None` if creating a new project to skip file search. | false |
```

Comment:
Based on this description I don't understand when I would use 'None'

Reply:
When creating a new project or skipping the file search option.